src/bussinessLogic/ManageData.java

```diff
@@ -0,0 +1,40 @@
+package bussinessLogic;
+
+import interfaces.ArrayManager;
+import interfaces.DataManager;
+import interfaces.MapManager;
+
+public class ManageData implements DataManager {
+    private final MapManager manageMap;
+    private final ArrayManager manageArray;
+    // int for calculating how many times a letter is displayed in the evaluation
+    // sequence
+    private int alphabetMultiplier = 1;
+
+    public ManageData(ArrayManager manageArray, MapManager manageMap) {
+        this.manageArray = manageArray;
+        this.manageMap = manageMap;
+    }
+
+    /**
+     * prepare all the data necessary for the evaluation phase
+     */
+    public void manageEvaluationData() {
+        manageArray.setEvaluationArray(alphabetMultiplier);
```

Comment:
use _this_ keyword to know that this is a reference to a private property.

Reply:
👍 
This is also valid for `managePracticeData()` method, am I right?

Reply:
Yes.